doc/source/_static/numpy.css

```
@@ -48,6 +47,7 @@ Nat Methods 8, 441 (2011). https://doi.org/10.1038/nmeth.1618
 #version_switcher_button[data-active-version-name*="dev"] {
   background-color: #E69F00;
   border-color: #E69F00;
+  color:#4A4A4A; /* numpy.org body color */
```

```suggestion
  color:#6C7A89; /* NumPy Slate Gray */
```

@noritada, thank you for sharing your thoughts! These are all valid concerns. For the version selection button in the dark mode, I suggest using NumPy Slate Gray (#6C7A89) from the NumPy color palette (https://github.com/numpy/numpy.org#numpy-color-palette).

---

@InessaPawson Thank you for the suggestion! I must have missed that color palette.
I have pushed the suggested color update.